vars/all.yml

```diff
@@ -0,0 +1,12 @@
+### Cluster ###
```

Comment:
I agree that this file is needed, but:

- It should be near `kubevirt.yml`, which is the playbook that will use it.
- We should include this file using `include_vars` from `kubevirt.yml`.
- This file shouldn't contain any variables that are not related to Kubevirt's deployment. You can place global 
  variables in `vars/global_vars.yml`, and I think that we shouldn't use this file often.

Reply:
The goal is for this file to be the top level interface for kubevirt-ansible, not just the kubevirt playbook.  Variables declared here like openshift_version and cluster will be used in multiple playbooks.  This should absorb ```vars/global_vars.yml```.

We can use ```include_vars``` in the playbook so we don't have to use ```-e@vars/all.yml```.